scripts/test_examples.js

```diff
@@ -38,37 +43,10 @@ function runExampleTests(exampleDirectory) {
     return;
   }
 
-  runCommand('yarn', 'install --pure-lockfile', exampleDirectory);
-  packages.forEach(pkg => {
-    const name = path.basename(pkg);
-    const directory = path.resolve(exampleDirectory, 'node_modules', name);
-
-    if (fs.existsSync(directory)) {
-      rimraf.sync(directory);
-      mkdirp.sync(directory);
-      // Using `npm link jest-*` can create problems with module resolution,
-      // so instead of this we'll create a proxy module.
-      fs.writeFileSync(
-        path.resolve(directory, 'index.js'),
-        `module.exports = require('${pkg}');\n`,
-        'utf8'
-      );
-      fs.writeFileSync(
-        path.resolve(directory, 'package.json'),
-        `{"name": "${name}", "version": "${VERSION}"}\n`,
-        'utf8'
-      );
-    }
-  });
-
-  // overwrite the jest link and point it to the local jest-cli
-  mkdirp.sync(path.resolve(exampleDirectory, './node_modules/.bin'));
-  runCommand('ln',
-    `-sf ${JEST_BIN_PATH} ./node_modules/.bin/jest`,
-    exampleDirectory
-  );
-
-  runCommand('yarn', 'test', exampleDirectory);
-}
+  if (INSTALL.indexOf(exampleName) !== -1) {
+    runCommand('yarn', '--production', exampleDirectory);
+  }
 
-examples.forEach(runExampleTests);
+  link(exampleDirectory, BABEL_JEST_PATH);
```

Comment:
Linking _after_ installing breaks `react-native` example. Moving this before install fixes the issue.